### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Feature_Link a Cluster-Link a Cluster_HD Insight_Keyboard: Focus on "Authentication Error Details" control is not visible

### ISSUE THREAD

**Issue by Amulya-95** [microsoft/azure-tools-for-java#3635]
[Check out Accessibility Insights!](https://github.com/microsoft/azure-tools-for-java/issues/url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHub Tags: #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #Keyboard; #MAS2.4.7

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.2
Build #IU-192.6603.28

**OS Details:**
OS: Windows (Version 1903, Build: 18362.356)
MAS Violated: MAS2.4.7

**Note: Similar issue is observed in** 
1.Link a Cluster-Link a Cluster_Livy Service
2.Link SQL Server big data cluster_Link SQL Server big data cluster
3.Link Cluster-Read only

**Repro Steps:**
1)Launch IntelliJ IDEA (Darcula, High contrast, IntelliJ theme)
2)Tab till Create New Project button and press enter
3)Tab till view-->Tool Window-->Azure Explorer and press enter
4)Tab till signin-->copy and open & sign in to azure
5)Tab till select button and press enter
6)Tab till HDInsight and open context menu by using keyboard key
7)Tab till link a cluster and press enter
8)Verify whether focus is visible on "Authentication Error Details" control or not

**Actual Result:**
Focus on "Authentication Error Details" control is not visible

**Expected Result:**
Focus should be visible  on "Authentication Error Details" control

**User Impact:**
Keyboard users will find difficulty in navigating if keyboard focus is not visible on the controls

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

**Attachment:**
[Focus on 'Authentication Error Details' control is not visible.pptx](https://github.com/microsoft/azure-tools-for-java/files/3656297/Focus.on.Authentication.Error.Details.control.is.not.visible.pptx)

**Comment by t-rufang** [comment]
Note:
1. When focus is at `Password` text field, press tab, focus is invisible

2. When we click ok, and link cluster failed. The `Authentication Error Details` folder expanded. But focus is invisible. 
![image](https://user-images.githubusercontent.com/32627233/65675925-95321080-e081-11e9-9286-6f3a6cf87681.png)

**Comment by t-rufang** [comment]
Fixed in #3679. Plese find the focus flow after the fix
![fix_issue_3635](https://user-images.githubusercontent.com/32627233/67481664-e63f1f80-f694-11e9-8f80-73451bd559aa.gif)

**Comment by v-lasiva** [comment]
Regressed the issue in the updated version of IntelliJ having the details,
Version: IntelliJ IDEA 2019.2.2

Focus is still not visible on "Authentication Error details dropdown".
As the issue is not resolved, Again re-activating the bug.
Added attachment for reference.

[Bug_Focus is not visible on authentication dropdown_Issue not fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3850226/Bug_Focus.is.not.visible.on.authentication.dropdown_Issue.not.fixed.pptx)

**Comment by t-rufang** [comment]
The "Authentication Error details" view is extended from `com.intellij.execution.impl.ConsoleViewImpl`, which is supported by IntelliJ. When we focus at a `ConsoleViewImpl`, the focus will be invisible, this should be fixed by IntelliJ team and it's out of the scope of our toolkit.

Please find the same issue when you run an application. See repro steps below:
1. Create a new configuration of type `Application`. Fill in the main class `sample.JavaSparkPi`
![image](https://user-images.githubusercontent.com/32627233/69040398-b3d0da00-0a28-11ea-9b5c-1d822cb94077.png)
2. Click `Run` button to run this application.
![image](https://user-images.githubusercontent.com/32627233/69040593-03170a80-0a29-11ea-96ee-a7be4eddb3e3.png)
3. This will trigger the console view implemented by IntelliJ. As you see, the focus at this console view is also invisible (no highlight border when we focus at the output console view).
![image](https://user-images.githubusercontent.com/32627233/69040671-2477f680-0a29-11ea-9aba-8e929074ae7c.png)

Therefore, this bug should be fixed by IntelliJ team and it's out of the scope of our toolkit.

**Comment by t-rufang** [comment]
- Importance of experience: Medium, users will not know whether the console view is on focus when they tab to this UI conponent.
- Quality of experience: Little, the blinking cursor will tell user the console view is currently focused.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by v-lasiva** [comment]
After discussing with SME, As they have approved closing the issue accordingly.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1738825
